packages/cssnano-preset-default/package.json

```diff
@@ -14,7 +14,7 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "css-declaration-sorter": "^4.0.1",
+    "css-declaration-sorter": "git://github.com/Siilwyn/css-declaration-sorter#phoenix",
```

Comment:
No need use git, some developers can't have git and it is failed

Reply:
Yes this is temporary so I can easily make changes to the module for now.

So I will publish version 5 before merging this PR, then I will change this!

Reply:
Hey @evilebottnawi I've published the new version!